src/kem.rs

```diff
@@ -49,19 +54,21 @@ impl From<ecdh::Error> for Error {
 pub enum PrivateKey {
     X25519([u8; 32]),
     P256([u8; 32]),
+    Kyber768([u8; kyber768::SECRET_KEY_SIZE]),
 }
 
 /// A KEM public key.
 pub enum PublicKey {
     X25519([u8; 32]),
     P256([u8; 64]),
+    Kyber768([u8; kyber768::PUBLIC_KEY_SIZE]),
 }
 
 /// Compute the public key for a private key of the given [`Algorithm`].
 pub fn secret_to_public(alg: Algorithm, sk: impl AsRef<[u8]>) -> Result<Vec<u8>, Error> {
```

Comment:
I'm not sure yet if we need this for Kyber. But the comment should say that it's only for EC algorithms.